src/main/java/com/team4/testingsystem/services/impl/QuestionServiceImpl.java

```
@@ -0,0 +1,23 @@
+package com.team4.testingsystem.services.impl;
+
+import com.team4.testingsystem.entities.Question;
+import com.team4.testingsystem.repositories.QuestionRepository;
+import com.team4.testingsystem.services.QuestionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class QuestionServiceImpl implements QuestionService {
+
+    private QuestionRepository questionRepository;
+
+    @Autowired
+    QuestionServiceImpl(QuestionRepository questionRepository) {
+        this.questionRepository = questionRepository;
+    }
+
+    @Override
+    public Question getQuestionById(Long id) {
+        return questionRepository.getQuestionById(id).orElse(null);
```

Please, throw errors instead of returning nulls